src/hooks/use-change-volume.ts

```diff
@@ -0,0 +1,73 @@
+import { useAndRequireContext } from "~/hooks/use-and-require-context.js";
+import { useEffect } from "react";
+import { ContextRef } from "~/containers/sound-provider/index.js";
+import { DEFAULT_VOLUME } from "~/constants/index.js";
+
```

Comment:
rename file to `volume.ts` since it's not just one hook but a bunch of hooks that relate to volume